internal/bridge/crunchybridgecluster/watches_test.go

```diff
@@ -0,0 +1,80 @@
+//go:build envtest
+// +build envtest
+
+/*
+ Copyright 2021 - 2024 Crunchy Data Solutions, Inc.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package crunchybridgecluster
+
+import (
+	"context"
+	"testing"
+
+	"gotest.tools/v3/assert"
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/crunchydata/postgres-operator/internal/testing/require"
+)
+
+func TestFindCrunchyBridgeClustersForSecret(t *testing.T) {
+	ctx := context.Background()
+	_, tClient := setupKubernetes(t)
+	require.ParallelCapacity(t, 1)
```

Comment:
TBH I added that simply because I saw it in our other `envtest` tests... I will try removing it.

Reply:
It looks like you removed `ParallelCapacity` rather than setting it to `0`. I see other new tests where it exists but is set to `0`. Do you have any idea what the difference between not running the function and setting it to `0` is?

Should we do one over the other?

Reply:
I think running it and setting it to `0` will signal that the test can be run in parallel. So i think we will want it to be set so that it can run in parallel with other tests.

https://github.com/CrunchyData/postgres-operator/blob/master/internal/testing/require/parallel.go#L29